libraries/AP_Motors/AP_MotorsHeli.h

```diff
@@ -245,7 +247,9 @@ class AP_MotorsHeli : public AP_Motors {
     struct heliflags_type {
         uint8_t landing_collective      : 1;    // true if collective is setup for landing which has much higher minimum
         uint8_t rotor_runup_complete    : 1;    // true if the rotors have had enough time to wind up
+#if AP_INVERTED_FLIGHT_ENABLED
```

Comment:
does this value show in logs and docs?

Reply:
Logging - not the first time I checked, and not now I just checked again (AFAICS)

And I'm not sure what documentation that might be, or how it might be relevant in that the documentation can still reference the flag, it might not be compiled into a random binary but iit isn't externally visible anyway.